scripts/characters/constructor.gd

```diff
@@ -343,7 +369,7 @@ func set_genealogy(person):
 		person.race_type = 4
 	
 	#Set Primary Race
-	if person == globals.player || person.unique != null || person.race in magic_races_array || rand_range(0,100) <= globals.expansionsettings.randompurebreedchance:
+	if person == globals.player || person.unique != null || person.race in magic_races_array || rand_range(0,100) <= globals.expansionsettings.randompurebreedchance || (person.race in uncommon_races_array && rand_range(0,100) <= globals.expansionsettings.randompurebreedchanceuncommon): #ralph2
```

Comment:
Gameplay change

Reply:
Actually doesn't need to be changed, the other condition is always false given the randompurebreedchanceuncommon set in Aric's is 0.

Reply:
Actually does rand_range potentially return 0?

Reply:
rand_range(0,100) <= 0 can be true, though the chances are basically 1 in a million.
rand_range(0,100) < 0 can never be true, but rand_range(0,100) < 100 can be false.
the simplest fix is to set it to -1 to disable it, rand_range(0,100) <= -1 can never be true.

Reply:
Gotcha, you fixed this one